### ISSUE TITLE
Focus directly not going to Send Button of Outlook once the taskpane is closed. [WCAG Compliance]

### ISSUE THREAD

**Issue by RaveendraReshma** [OfficeDev/office-js#4323]
1. We are developing office web Add in for outlook using event based (Smart Alert) .The Addin is generated using office yo generator Task Pane add-in using the React framework.
2. We are getting the recipient data [ to/cc/bcc ] using Office.context.mailbox.item and we are rendering the same recipients onto the task pane.
3. We have used soft block on OnSendHandler launch event.

## Your Environment

- Platform : Office on the web
- Host : Outlook [OWA] & New Outlook [Desktop]
- Office version number : Version 2311 Build 16.0.17029.20140
- Operating System : Windows
- Browser : Chrome & Edge

## Expected behavior
1. Once the task pane is closed the focus should go back directly to Send Button of Outlook so that user can send the mail out.

## Current behavior

1. When ever visually impaired user closes the task pane and tries to use tab key from keyboard to navigate to send button of Outlook. The focus directly is not going back to send button instead it starts from the starting element of the page.

## Steps to reproduce

Use Only Tab, Enter/Space Bar for repro

1. Using Tab key from keyboard Navigate to New Email and press Enter.

![image](https://github.com/OfficeDev/office-js/assets/145095553/9572d6fc-5981-4585-9422-28efdd2b044a)

2. Add Recipients in To/Cc/Bcc Section. Once entered, using [Shift + Tab] Keys navigate back to Send Button of Outlook and press Enter.

![image](https://github.com/OfficeDev/office-js/assets/145095553/1634448b-fd0d-4360-ba29-85532f1b0f0f)

3. Once the smart alert appears. Using Tab Key Navigate to Send Button and press Enter.

<img width="400" alt="image" src="https://github.com/OfficeDev/office-js/assets/145095553/14bb1dd0-fcc1-4027-9a1b-31273d9b3b6e">

4. Now the task pane will be shown. Now once user verifies the recipients using screen reader and navigates to cross icon of the task pane and press Enter the task pane will be closed.

<img width="357" alt="image" src="https://github.com/OfficeDev/office-js/assets/145095553/84d8a8a5-d176-4202-84e7-665dd26cac6a">

5. Then the moment task pane is closed the focus is not going back directly to send button of Outlook instead the focus is going to the first element in the page.

<img width="490" alt="image" src="https://github.com/OfficeDev/office-js/assets/145095553/f7dd66fe-a48a-49cc-bd67-df93b0dc0a6a">

## Context

So basically every time user wants to send the mail he/she have to navigate through entire screen from the start to reach the send Button of Outlook.

**Comment by rajjha-msft** [comment]
Hey @RaveendraReshma 

Its not a repro for me. Can you attach a repro video around the same ?
